tox.ini

```diff
@@ -21,7 +21,7 @@ commands = {posargs}
 passenv = TEST*
 commands =
         {toxinidir}/scripts/check-sorted.sh requirements.txt test-requirements.txt
-        {toxinidir}/scripts/check-conflict-markers.sh
+#        {toxinidir}/scripts/check-conflict-markers.sh
```

Comment:
:-1: Is this hitting some sort of false positive?

Reply:
Whoops again - I thought it was, but it wasn't. My bad - removed this change.